ShenkinStore/Models/ShoppingCart.cs

```diff
@@ -0,0 +1,164 @@
+﻿using Microsoft.AspNetCore.Http;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+
+
+namespace ShenkinStore.Models
+{
+    public class ShoppingCart
+    {
+        // Database initiatl
+        ShenkinContext db = new ShenkinContext();
+        // Data for Recognation
+        public string ShoppingCartId { get; set; }
+
+        public const String CartSessionKey = "cartId";
+
+       public static Dictionary<int, int> productsAmount = new Dictionary<int, int>();
```

Comment:
I all those can be in 1 dictionary.
Dictionary<userId(int), Dictionary<Product, amount(int)>>.
And if you want the total just get all the prices of the products in a specific cart with lamda (if you need help LMK).